api/jobs.yaml

```diff
@@ -379,6 +382,11 @@ definitions:
         description: >
           If specified, returns only jobs containing labels with exact string
           matches to each of these labels.
+      id:
```

Comment:
This seems a little weird to query on. We already have the GET `jobs/{id}` endpoint to get a job with a particular ID. A user could see a particular job by going to: `URL:4200/jobs/{id}`. Is this a feature that was requested?